### PR TITLE
docs: Mechanical Edits

### PR DESCRIPTION
**Pull request by syedjawadakhtar**
Shifted mechanical content from index.md to mechanical README.md

### CODE REVIEW

File: docs/computer/Acoustics.md [hunk 1]
```diff
@@ -29,8 +29,14 @@ Points would be awarded as follows. If the AUV successfully causes the ball to d
 flare with the pinger: 40 Points
 [This task from the [rulebook of SAUVC 2018](https://sauvc.org/2018/)]
 
-![yellow_Flare_animated](static/flare_Animation.png)
-![yellow_Flare_underwater](static/yellow_Flare_water.png)
+
+| ![yellow_Flare_animated](static/flare_Animation.png) |
```
Comment (pnijhara) [inline docs/computer/Acoustics.md:33]: These changes were already there in acoustics?

**Review by pnijhara — CHANGES_REQUESTED** [review]

File: docs/computer/Acoustics.md (hunk 1 above)
Comment (syedjawadakhtar) [reply docs/computer/Acoustics.md:33]: Yup it was there.
Couldn't get it why it added this file again.

Comment (pnijhara) [reply docs/computer/Acoustics.md:33]: I think you haven't pulled the changes before sending the PR.
Can you make another branch and pull the changes before making your own changes?
Then send another PR.

Comment (syedjawadakhtar) [reply docs/computer/Acoustics.md:33]: I have pulled the repo and made a new branch `j-mech-patch` and made a PR again. 
Please check.

**Comment by pnijhara** [comment]
@syedjawadakhtar I am closing this PR.

**Pull request closed without merging**